web/src/stream_list.js

```diff
@@ -597,6 +597,11 @@ export function initialize() {
     build_stream_list();
     update_subscribe_to_more_streams_link();
     set_event_handlers();
+
+    // register a callback function in narrow.js so that we 
+    // wont need narrow.js to import stream_list.js module
+    // which induce cyclic import.
+    narrow.register_stream_list_callback(handle_narrow_deactivated);
```

Comment:
It should be `handle_narrow_activated` here instead of `handle_narrow_deactivated`. You have also made this mistake in your commit message.

Reply:
You're right. Thanks for pointing out!